_posts/2012-02-28-odata-service-validation-tool-update-ground-work-for-v3-support.html

```diff
@@ -24,5 +24,5 @@
 </ul>
 <p>This update brings the total number of rules in the validation tool to <strong>152.</strong> You can see the list of the active rules <a href="https://services.odata.org/validation/roadmap.htm#rules">here</a> and the list of rules that are under development <a href="https://services.odata.org/Validation/RuleSet/Rules?$filter=Status%20eq%20%27Approved%27">here</a>.</p>
 <p>Moreover, with this update the validation tool is positioned to support rules for v3 services for OData V3 payload validation.</p>
-<p>Keep on  validating and let us know what you think either on the <a href="https://www.odata.org/join-the-odata-discussion/">mailing list</a> or on the <a href="http://odatavalidator.codeplex.com/discussions">discussions page</a> on the Codeplex site.</p>
+<p>Keep on  validating and let us know what you think either on the <a href="/join-the-odata-discussion/">mailing list</a> or on the <a href="http://odatavalidator.codeplex.com/discussions">discussions page</a> on the Codeplex site.</p>
```

Comment:
Broken link http://odatavalidator.codeplex.com/discussions

Reply:
@KenitoInc Fixing dead links that point to external resources was beyond the scope of this pull request. In any event, I didn't have a valid alternative I could replace them with.

Reply:
if these links are broken.. Can we replace them with `#` instead of leaving them as broken links..

Reply:
@ElizabethOkerio I was avoiding making a "material change" in this PR. We could replace the broken links with # but that decision might require more inputs/consideration. We could also consider getting rid of posts in the archived category that reference broken links. Either way I'd rather defer those decisions for another PR. The redesign PR is a more appropriate opportunity to deal with the broken links